sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -83,13 +82,13 @@ public KeyVaultKeyStore() {
         String tenantId = System.getProperty("azure.keyvault.tenantId");
         String clientId = System.getProperty("azure.keyvault.clientId");
         String clientSecret = System.getProperty("azure.keyvault.clientSecret");
-        keyVault = new KeyVaultClient(keyVaultUri, tenantId, clientId, clientSecret);
+        keyVaultClient = new KeyVaultClient(keyVaultUri, tenantId, clientId, clientSecret);
```

Comment:
```suggestion
        keyVaultClient = new KeyVaultClient(keyVaultUrl, tenantId, clientId, clientSecret);
```
All occurrences, really :)